### PR TITLE
Fix fixed size arrays

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/880
Result of https://github.com/gtk-rs/gir/pull/881

cc @sdroege , @GuillaumeGomez 

No other meaningful changes except for not included removal empty line in "Cargo.toml"s (as I remember @GuillaumeGomez don't add it to regen anyway).

### CODE REVIEW

File: gtk-sys/src/lib.rs [hunk 1]
```diff
@@ -14222,14 +14221,8 @@ pub struct GtkStyle {
...
     pub ythickness: c_int,
-    pub background: *mut [*mut cairo::cairo_pattern_t; 5],
-    pub attach_count: c_int,
-    pub visual: *mut gdk::GdkVisual,
-    pub private_font_desc: *mut pango::PangoFontDescription,
-    pub rc_style: *mut GtkRcStyle,
-    pub styles: *mut glib::GSList,
-    pub property_cache: *mut glib::GArray,
-    pub icon_factories: *mut glib::GSList,
+    _truncated_record_marker: c_void,
```
Comment (GuillaumeGomez) [inline gtk-sys/src/lib.rs]: `background` should be there no?

Comment (EPashkin) [reply gtk-sys/src/lib.rs]: It really incomplete 
`#[repr(C)] pub struct cairo_pattern_t(c_void);`

**Comment by GuillaumeGomez** [comment]
I guess it's time for full regen @EPashkin ? :)

**Comment by EPashkin** [comment]
Almost, I missed warning https://github.com/gtk-rs/gir/pull/882

**Comment by EPashkin** [comment]
Updated.
Again without removal of empty line in Cargo.toml but this can cause CI fail

**Comment by GuillaumeGomez** [comment]
We'll check that later on. Thanks!

**Pull request merged**